engine/schema/src/com/cloud/offerings/NetworkOfferingVO.java

```diff
@@ -142,6 +142,9 @@
     @Column(name="supports_streched_l2")
     boolean supportsStrechedL2 = false;
 
+    @Column(name="supports_public_access")
+    boolean supportsPublicAccess = false;
+
```

Comment:
In our SDN solution, shared networks are not always reachable.
If a shared network is routable, in the same way a public network is, 
this flag will let the Nuage VSP know that it should distribute routes for that shared network.